data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -1044,7 +1046,7 @@
             unrenamable=yes
             profile=portraits/elyssa.webp
             gender=female
-            experience=15
+            experience={ON_DIFFICULTY 17 16 15}
```

Comment:
Why?

Reply:
to make it easier to level her up earlier; I think I had some cases where she died on me just a few XP away from advancing...

Reply:
For a unit that needs 125 XP, a difference of just 1XP sounds like it would be specific to the RNG that you got.

Reply:
ok so maybe `{ON_DIFFICULTY 35 25 15}` then?

Reply:
Reminder that XP now scales with difficulty. I get her almost leveled on nightmare with 200% xp mod. Is there some cool interaction if she levels? If not there should be I think.

Reply:
> Reminder that XP now scales with difficulty.

It does? When did that happen?

Reply:
Well I already said that a lot happened. Perhaps it was not clear enough. You should play it or check commit history. Or both.

Reply:
Why should everyone that needs to know what changed have to browse through the commit history?

Reply:
I would assume that knowing the up to date version of content that you want to change is the bare minimum when it comes to making changes. You also don't have to do that, checking scenarios and the game / content i usually s enough as I also stated before.

Reply:
And how exactly are we supposed to know and be sure we know them all, as you are not sharing it in a descriptive way?

It is obvious that the author of this PR has played it as he did actually implement changes to make it workable to him.

Reply:
I'm not saying this for whatever your comment is about I'm saying that because author admitted that these were changes for 1.14 which is pretty much 2 major versions in the past bruh. So what you are saying is obvious... Is not true since author stated that himself earlier. XD

Reply:
Also if even easier version is needed then perhaps add 4th difficulty.

Reply:
> I'm not saying this for whatever your comment is about I'm saying that because author admitted that these were changes for 1.14 which is pretty much 2 major versions in the past bruh.

Just 1; 1.18 still isn't out yet. Also the fact that I was able to get these diffs to apply at all shows that things haven't changed *THAT* much since 1.14...

Reply:
Because the scenarios werent reworked, but balance and units were.

Reply:
@Hejnewar is this the part where you meant "XP maybe 25 on all difficulties"? I prefer more variation between difficulties rather than less, so how about one of these:
- `{25 20 15}`
- `{35 25 15}`
- `{30 25 20}`
- `{27 26 25}`
- `{26 25 24}`
- `{25 24 23}`
- `{25 23 21}`
- `{25 22 19}`
- `{25 21 17}`